### DISCUSSION TITLE
How to trigger RPC function with CRON job?

### DISCUSSION THREAD

**Discussion by orenlande** [supabase/supabase#16372] [Questions] [answered]
Hello, 
I have a custom function `insert_daily_lingo` in the public schema.

In the cron table, the cell `command` looks like this - `select rpc('insert_daily_lingo');`, but it fails with the following error:
```
ERROR:  function rpc(unknown) does not exist
LINE 2:     select rpc('insert_daily_lingo');
                   ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
```

I also tried to use `select rpc('public.insert_daily_lingo');` but the result remains the same. 
When I'm inserting the actual function content (the full SQL) it works, but I feel like it's the wrong way to approach. Anyone knows how to trigger custom RPC with crons?

Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
You need to use SQL and rpc is not part of that (it is the client api way to call a function).

So either `select insert_daily_lingo();` or `perform insert_daily_lingo();` if you are not using the result.
